app/views/provider_interface/rejections/check.html.erb

```diff
@@ -23,17 +23,36 @@
         <p class="govuk-body"><%= @interview_cancellation_presenter.text %></p>
       <% end %>
 
-      <div class="govuk-warning-text">
-        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-        <strong class="govuk-warning-text__text">
-          <span class="govuk-warning-text__assistive"></span>
-          The candidate will be sent an email to tell them why you rejected their application.
-        </strong>
-      </div>
-
       <% if rbd_application_with_no_feedback? %>
         <%= f.govuk_submit 'Give feedback' %>
       <% else %>
+        <p class="govuk-body">The candidate will be sent this email:</p>
+        <div class="app-email-preview">
+          <p class="govuk-body">Dear <%= @application_choice.application_form.first_name %>,</p>
+
+          <p class="govuk-body">Thank you for your application to study <%= @application_choice.current_course_option.course.name_and_code %> at <%= @application_choice.current_course_option.course.provider.name %>.</p>
+
+          <p class="govuk-body">On this occasion, the provider is not offering you a place on this course.</p>
+
+          <p class="govuk-body">They’ve given the following feedback to explain their decision:</p>
+
+          <div class="govuk-inset-text">
+            <% @wizard.flattened_reasons.each do |reason| %>
+              <% next if reason.nil? %>
+              <% if reason.details.present? %>
+                <% if reason.details.text.empty? %>
+                  <p class="govuk-body"><%= reason.label %></p>
+                <% else %>
+                  <p class="govuk-body"><%= reason.label %>:</p>
+                  <p class="govuk-body"><%= reason.details.text %></p>
+                <% end %>
+              <% else %>
+                <p class="govuk-body"><%= reason.label %></p>
+              <% end %>
+            <% end %>
+          </div>
+          <p class="govuk-body"> Contact <%= @application_choice.current_course_option.course.provider.name %> if you would like to talk about their feedback.</p>
+        </div>
```

Comment:
Can we somehow use the exact same partial/component as is used in the email to candidates, so that this stays in sync with whatever changes we might make to the email?  Otherwise they’re bound to accidentally diverge?

Reply:
Yeah this is a completely valid concern. 

The reason I haven't used the [reasons_for_rejection.text.erb](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/views/candidate_mailer/_reasons_for_rejection.text.erb) partial is because of the rejection data format. By the time this partial is used in the email the R4R has been written to the DB, and is associated with the application choice. During the rejection flow, the R4R is being held in memory for the purpose of the RejectionWizard and requires a bit of it's own formatting to get it into an array that's useable for the above.

Happy to think further on this though!

Reply:
@JR-G ah, guess that makes it more complicated. Is there a way to extract the view into some kind of component which can either read the rejection reasons from the database or read it from the rejection reasons held in memory?

If not, I guess we have to live with this and somehow remember to keep them in sync?

Reply:
@carlosmartinez , @stevehook  & @tomas-stefano any thoughts on this ☝️  ?

Reply:
Also worth mentioning there's 5 different version of a rejection email, that uses our partials (not all necessary applicable here):
<img width="713" alt="image" src="https://user-images.githubusercontent.com/47917431/223696278-ecbb9962-07a4-417a-a0b2-5eaa57ca8ddc.png">